server/server.js

```diff
@@ -103,6 +103,7 @@ function server(serviceManager, options) {
     // take care to not callback until any actions are complete.
     if (!callback) {
       callback = function(err) {
+        // XXX(sam) shouldn't err of an err-first already be an Error() object?
         if (err) throw Error(err);
```

Comment:
@kraman I think the `Error: Error: ...` we saw in meshctl demo might be from here, or code like this.